sklearn/model_selection/_search.py

```diff
@@ -1411,7 +1411,24 @@ class RandomizedSearchCV(BaseSearchCV):
         A generator over parameter settings, constructed from
         param_distributions.
 
+
+    Examples
+    --------
+    >>> from sklearn import linear_model, datasets
+    >>> from sklearn.model_selection import RandomizedSearchCV
+    >>> from scipy.stats import  uniform
+    >>> iris = datasets.load_iris()
+    >>> logistic = linear_model.LogisticRegression(multi_class='auto',
+    ...                                             penalty='l2', max_iter=200)
+    >>> hyperparameters = dict(C=uniform(loc=0, scale=4),
+    ...                          solver=['newton-cg', 'lbfgs'])
+    >>> clf = RandomizedSearchCV(logistic, hyperparameters,
+    ...                            random_state=0, cv=5)
+    >>> search = clf.fit(iris.data, iris.target)
+    >>> search.best_params_
+    {'C': 2.195254015709299, 'solver': 'lbfgs'}
```

Comment:
Use ellipsis
```suggestion
    {'C': 2.1..., 'solver': 'lbfgs'}
```

Reply:
Hi @glemaitre 
When I truncate the `C` parameter value my test fails.

Reply:
Got it

Reply:
You need to use the `...` which is the ellipsis which will match whatever number after `2.1`